### PR TITLE
Make travis to static and spec stages with default mechanism but using PDK

### PR DESCRIPTION
**Pull request by timidri**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,26 +3,28 @@ os: linux
...
-  - bundle -v
+# before_install:
+#   - bundle -v
+#   - rm -f Gemfile.lock
+#   - "# Update system gems if requested. This is useful to temporarily workaround troubles in the test runner"
+#   - "# See https://github.com/puppetlabs/pdk-templates/commit/705154d5c437796b821691b707156e1b056d244f for an example of how this was used"
+#   - "# Ignore exit code of SIGPIPE'd yes to not fail with shell's pipefail set"
+#   - '[ -z "$RUBYGEMS_VERSION" ] || (yes || true) | gem update --system $RUBYGEMS_VERSION'
+#   - gem --version
+#   - bundle -v
```
Comment (mcka1n) [inline .travis.yml]: Since this block is not used anymore, should we remove it?

```suggestion

```

File: .travis.yml [hunk 2]
```diff
@@ -3,26 +3,28 @@ os: linux
...
 script:
-  - 'pdk validate'
-bundler_args: --without system_tests
-rvm:
-  - 2.7.6
+  - 'pdk $CHECK'
+# bundler_args: --without system_tests
+# rvm:
+#   - 2.5.7
+#   - 2.7.6
```
Comment (mcka1n) [inline .travis.yml]: This other section can be deleted now :) 

P.S: I'm curious if the below GitHub suggestion will do the job:

```suggestion

```

File: .travis.yml [hunk 3]
```diff
@@ -32,9 +34,17 @@ jobs:
   fast_finish: true
   include:
     -
-      env: PUPPET_GEM_VERSION="~> 7.16.0" CHECK=parallel_spec
-      rvm: 2.7.6
+      env: CHECK="validate --puppet-version=7.4.1" # PE 2021.7
+      stage: static
+      # rvm: 2.7.6
```
Comment (mcka1n) [inline .travis.yml]: Are these `rvm` versions still needed?

[Ln. 43](https://github.com/puppetlabs/puppetlabs-puppet_bolt_server/pull/8/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R43)

[Ln. 47](https://github.com/puppetlabs/puppetlabs-puppet_bolt_server/pull/8/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R47)

File: manifests/init.pp [hunk 4]
```diff
@@ -25,12 +25,12 @@
   file { 'puppet-token':
     ensure  => file,
     path    => '/root/.puppetlabs/token',
-    content => $puppet_token.unwrap,
+    content  => $puppet_token.unwrap,
```
Comment (mcka1n) [inline manifests/init.pp]: I'm curious if this was an intended change. It seems the indentation is off now, isn't it?

**Review by mcka1n — CHANGES_REQUESTED** [review]
Hey @timidri thanks for making Travis work again!

I checked the Travis builds, look good to me! 👍  but I left some questions on this PR.

Let me know what you think :)

File: manifests/init.pp (hunk 4 above)
Comment (timidri) [reply manifests/init.pp]: This was intended to see if Travis picks this up. I had a suspicion that static tests don't work properly.

File: .travis.yml (hunk 3 above)
Comment (timidri) [reply .travis.yml]: I don't think so. I'll remove them.

**Comment by timidri** [comment]
Hi Edwin, thanks for reviewing, I made the requested changes and pushed.

Dimitri Tischenko
Solutions Architect | EMEA
mobile: +31 654 626664
***@***.******@***.***> | @timidri
slack.puppet.com<http://slack.puppet.com/> | #puppet

________________________________

**Review by mcka1n — APPROVED** [review]

**Comment by mcka1n** [comment]
LGTM 👍

**Pull request merged**